File: ticketProject7-CrossPlatform/TS-crossp/networking/response/LoginResponse.cs
using System;
namespace networking.protocol
{

	using SellerDTO = networking.dto.SellerDTO;

	[Serializable]
	public class LoginResponse : Response
	{
		private SellerDTO seller;

		public LoginResponse(SellerDTO seller)
		{
			this.seller = seller;
		}

		public virtual SellerDTO Seller
		{
			get
			{
				return seller;
			}
		}
	}

}

File: ticketProject6-RMI/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/LoginResponse.java
package network.objectprotocol;

import network.dto.SellerDTO;

public class LoginResponse implements Response {
    private SellerDTO seller;

    public LoginResponse(SellerDTO seller) {
        this.seller = seller;
    }

    public SellerDTO getSeller() {
        return seller;
    }
}
